fedbiomed/researcher/secagg/_secagg_context.py

```diff
@@ -560,14 +669,9 @@ def __init__(self, parties: List[str], experiment_id: str, secagg_id: Union[str,
             FedbiomedSecaggError: bad argument type or value
         """
         super().__init__(parties, experiment_id, secagg_id)
-
-        if len(parties) < 2:
-            raise FedbiomedSecaggError(
-                f'{ErrorNumbers.FB415.value}: LOM, bad parameter `parties` : {parties} : need '
-                'at least 2 nodes for secure aggregation')
-
-
         self._element = SecaggElementTypes.DIFFIE_HELLMAN
+        self._raise_if_missing_parties(parties, required_nb_parties=2)
+
```

Comment:
Hi @ybouilla, 

I think a nice way of implementing it could be having a class attribute `_min_num_parties`, and check it in base `SecaggContext.__init__`, as it is done in SecaggSetup classes on the node side.

Reply:
I agree with you, but in the previous implementation, there were no `_mion_num_parties`. I can implement it though!